chia/cmds/init_funcs.py

```diff
@@ -313,7 +313,15 @@ def chia_full_version_str() -> str:
     return f"{major}.{minor}.{patch}{dev}"
 
 
-def chia_init(root_path: Path):
+def chia_init(root_path: Path, skip_check_keys: bool = False):
```

Comment:
This will force people to write `chia_init(path, skip_check_keys=True)` instead of `chia_init(path, True)`

```suggestion
def chia_init(root_path: Path, *, skip_check_keys: bool = False):
```